app/src/sharedTest/java/org/oppia/app/utility/TabMatcher.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.app.utility
+
+import android.view.View
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+import androidx.test.espresso.PerformException
+import com.google.android.material.tabs.TabLayout
+import org.hamcrest.Matcher
+
+/**
+ * @see <a href="https://stackoverflow.com/q/49626315">TabMatcher</a>
+ * This class mainly provides a custom matcher to test whether the current tab title is correctly selected in TabLayout. */
```

Comment:
We should format KDocs consistently, and it's helpful to list the purpose of the class before the reference. Suggest:

```kotlin
/**
 * A custom matcher to test whether the current tab title is correctly selected in a [TabLayout].
 *
 * Reference: https://stackoverflow.com/q/49626315
 */
```

(Note that KDoc uses Markdown rather than HTML, but in this case I think we can just reference the URL directly rather than naming it ).

Reply:
Resolved